File: pep/lec1/table.cpp
#include <iostream>
using namespace std;
int main(){
    cout<<"Enter the table number and upto";
    int n,a ;
    cin>>n>>a;
    for(int i =1;i<=a;i++){
        cout<<n<<" * "<<i<<" = "<<n*i<<endl;
    }

    return 0;
}

File: pep/Arrays/basic/lec1/table.java
import java.util.Scanner;
public class table{
    public static Scanner scn = new Scanner(System.in);
    public static void main(String[] args){
        System.out.println("Enter the number and upto");
        int n = scn.nextInt();
        int a = scn.nextInt();
        for(int i =1;i<a;i++){
            System.out.println( n*i);
        }

    }
}